muspinsim/experiment.py

```diff
@@ -324,6 +331,15 @@ def Htot(self):
     def p_operator(self):
         return self._system.muon_operator(self.p)
 
+    def _apply_results_function(self, results: ArrayLike, variables: dict):
```

Comment:
```suggestion
    def apply_results_function(self, results: ArrayLike, variables: dict):
```
See other comment, since this is called in `FittingRunner` the leading underscore may be misleading (it mislead me, at least).